content/integrations/postgresql.md

```diff
@@ -42,63 +40,87 @@ When it prompts for a password, enter the one used in the first command.
-* `password` (Optional)
-* `dbname` (Optional) - Name of the database you want to monitor
-* `ssl` (Optional) - Defaults to False. Whether to use SSL to connect.
-* `tags` (Optional) - Dictionary of tags
-* `relations` (Optional) - Dictionary of per-relation (table) metrics that you want to track. Each relation generates 10 metrics + 10 metrics per index.
-
-  By default all schemas are included. To track relations from specific schemas only, use the following syntax:
+* `username` (Optional) - The user account used to collect metrics, set in the [Installation section above](#installation)
+* `password` (Optional) - The password for the user account.
+* `dbname` (Optional) - The name of the database you want to monitor.
+* `ssl` (Optional) - Defaults to False. Indicates whether to use an SSL connection.
+* `use_psycopg2` (Optional) - Defaults to False. Setting this option to `True` will force the Datadog Agent to collect PostgreSQL metrics using psycopg2 instead of pg8000. Note that pyscopg2 does not support SSL connections.
+* `tags` (Optional) - A list of tags applied to all metrics collected. Tags may be simple strings or key-value pairs.
+* `relations` (Optional) - By default, all schemas are included. Add specific schemas here to collect metrics for schema relations. Each relation will generate 10 metrics and an additional 10 metrics per index. Use the following structure to declare relations:
 
       relations:
-        - relation_name: another_table
+        - relation_name: my_relation
           schemas:
-            - public
-            - prod
+            - my_schema_1
+            - my_schema_2
 
 * `collect_function_metrics` (Optional) - Collect metrics regarding PL/pgSQL functions from pg_stat_user_functions
-* `collect_count_metrics` (Optional) - Collect count metrics, default value is True for backward compatibility but they migth be slow, suggested value is False.
+* `collect_count_metrics` (Optional) - Collect count metrics. The default value is `True` for backward compatibility, but this might be slow. The recommended value is `False`.
+
+
+## Custom metrics
+
+Datadog will generate PostgreSQL metrics from custom query results. For each custom query, four components are required: `descriptors`, `metrics`, `query`, and `relation`.
+
+* `descriptors` is used to add tags to your custom metrics. It's a list of lists each containing 2 strings. The first string is for documentation purposes and usually used to make clear what we are getting from the query. The second string will be the tag name. Note that instance tags will automatically be added to that list.
+* `metrics` are key-value pairs where the key is the query column iname or column function and the value is a tuple containing the Datadog custom metric name and metric type (`RATE`, `GAUGE`, or `MONOTONIC`). In the example below, the results of the sum of the `idx_scan` column will appear in Datadog  with the metric name `postgresql.idx_scan_count_by_schema`.
```

Comment:
> Datadog custom metric name

omit 'Datadog'? This name is what we the user are choosing, yet 'Datadog custom metric name' makes it sound somehow like some reserved Datadog-custom thing (whatever that would mean). I suppose the intent here was to say 'this will be the name of your metric when you look for it in the Datadog app'? If so, I think that's pretty obvious.